python_modules/dagster/dagster/core/execution.py

```diff
@@ -258,7 +260,25 @@ def _validate_environment(environment, pipeline):
             )
 
 
-def _create_config_value(config_type, config_input):
+def create_compute_node_graph(pipeline, yaml_config):
```

Comment:
This function is what makes me unhappy about passing strings in GraphQL :( Tbh, I'd rather have yaml be implementation detail, not something that ingrained into our system.

Reply:
per discussion on slack, going to have this be the JSON/GenericScalar/InputAnyType